public/app/core/components/NavBar/NavBarMenuItem.tsx

```diff
@@ -14,6 +14,7 @@ export interface Props {
   url?: string;
   adjustHeightForBorder?: boolean;
   isMobile?: boolean;
+  ExtraContent?: ComponentType;
```

Comment:
is `ExtraContent` needed anymore? i don't see anything passing it in

Reply:
Good catch, missed that one.